src/main/kotlin/no/nav/syfo/infrastructure/kafka/KandidatStatusProducer.kt

```diff
@@ -60,6 +94,7 @@ data class KandidatStatusRecord(
 data class StatusDTO(
     val value: SenOppfolgingStatus,
     val isActive: Boolean,
+    val createdAt: OffsetDateTime,
 )
```

Comment:
Lurer på om vi her kan begynne å lure på hva som egentlig er `createdAt` etterhvert🤔 For å få det litt tydeligere hva som gjelder kandidat og hva som gjelder vurdering, kunne det gått an å gjøre noe sånn👇😊
```kotlin
KandidatStatusRecord(
                uuid = kandidat.uuid,
                createdAt = kandidat.createdAt,
                personident = kandidat.personident.value,
                status = StatusDTO(
                    value = kandidat.status,
                    isActive = kandidat.status.isActive,
                ),
		sisteVurdering = NoeVurderingDTO(
			veilederIdent = …,
			createdAt = …,
			…,
		),
            )
```

Reply:
Jess, det blir bedre :+1: